great-idea/index.html

```diff
@@ -16,8 +16,82 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
+    <nav class="nav">
+      <a href="">Services</a>
+      <a href="">Product</a>
+      <a href="">Vision</a>
+      <a href="">Features</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </nav>
+
+    <section class="top">
+      <div class="top-left-section">
+        <h1>
+          Innovation On Demand
```

Comment:
i'd like to see a break or two in here so that all the words are on different lines.